changelogs/fragments/clarify_error_message.yml

```diff
@@ -1,2 +1,2 @@
 bugfixes:
-    - Fix an atomic_move error that is 'true', but  misleading. Now we show all 3 files involved and clarify what happened.
+    - Fix an atomic_move error that is 'true', but misleading. Now we show all 3 files involved and clarify what happened.
```

Comment:
This file exists in devel. The fix needs to be applied there and backported in a separate PR.